code/modules/gear_presets/uscm.dm

```diff
@@ -780,8 +780,6 @@
 	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/general/large(new_human), WEAR_R_STORE)
 	new_human.equip_to_slot_or_del(new /obj/item/device/binoculars(new_human), WEAR_L_HAND)
 
-	new_human.back.pickup()
```

Comment:
Both radio backpack presets did this some reason but yeah it would throw a runtime - is there anything broken with the radio backpack if you ckey into these mobs now that they don't perform pickup?

Reply:
I honestly didn't even bother checking what the breakable might be because the thing runtimes anyway and it's been like that for 5 months -- but it's true it might still trigger ID Changes in `/obj/item/storage/backpack/marine/satchel/rto/pickup`

Reply:
https://github.com/cmss13-devs/cmss13/pull/727 is the origin of that pickup call.

We probably need that pickup function to fire ever a mind enters this mob (e.g. Login?) and or you can try passing new_human as the argument for pickup to see if that then properly registers the signals.

Reply:
should be fixed, just made phone setting happen on equipped() aswell